gdpr/vendorlist-fetching.go

```diff
@@ -71,7 +79,7 @@ func vendorListURLMaker(version uint16) string {
 // The goal here is to update quickly when new versions of the VendorList are released, but not wreck
 // server performance if a bad CMP starts sending us malformed consent strings that advertize a version
 // that doesn't exist yet.
-func newOccasionalSaver(timeout time.Duration) func(ctx context.Context, client *http.Client, url string, saver saveVendors) {
+func newOccasionalSaver(timeout time.Duration, cTFVer int) func(ctx context.Context, client *http.Client, url string, saver saveVendors) {
 	lastSaved := &atomic.Value{}
```

Comment:
Function `newOccasionalSaver` seems to not be passing `cTFVer int` to the `saveOne` call. In order to make sure
we are not missing anything the compiler wouldn't catch, I recommend merging the `prebid/go-gdpr` changes
first before commiting this one.
```
func newOccasionalSaver(timeout time.Duration, cTFVer int) func(ctx context.Context, client *http.Client, url string, saver saveVendors) {
	lastSaved := &atomic.Value{}
	lastSaved.Store(time.Time{})

	return func(ctx context.Context, client *http.Client, url string, saver saveVendors) {
		now := time.Now()
		if now.Sub(lastSaved.Load().(time.Time)).Minutes() > 10 {
			withTimeout, cancel := context.WithTimeout(ctx, timeout)
			defer cancel()
			saveOne(withTimeout, client, url, saver) //<-- missing `cTFVer` that depicts the GDPR version and determines the URL to send the http request to
			lastSaved.Store(now)
		}
	}
}
```

Reply:
New go-gdpr library updated and all the tests and issues fixed